rubrics/tool/src/main/frontend/imports/sakai-rubric-grading-comment.html

```diff
@@ -12,6 +12,7 @@
     <div class="popover criterion-edit-popover left">
       <div class="arrow"></div>
       <div class="popover-title">
+        <label class="criterion-title" for=""><sr-lang key="comment_for_criterion" values='["{{criterion.title}}"]'>Comment for Criterion</sr-lang></label>
```

Comment:
Doubt here @brdebr , when you want to define two or three values, is this the right syntax?
values='["{{criterion.title}}", "{{criterion.title2}}", "{{criterion.title3}}"]'